scripts/tools/RegisterPublishedDeliveryWebhook.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoPublishing\scripts\tools;
+
+
+use Exception;
+use oat\oatbox\event\EventManager;
+use oat\oatbox\extension\script\ScriptAction;
+use oat\oatbox\reporting\Report;
+use oat\oatbox\service\ConfigurableService;
+use oat\oatbox\service\ServiceManagerAwareTrait;
+use oat\tao\model\webhooks\configEntity\Webhook;
+use oat\tao\model\webhooks\WebhookEventsService;
+use oat\tao\model\webhooks\WebhookRegistryManager;
+use oat\tao\model\webhooks\WebhookRegistryManagerInterface;
+use oat\taoPublishing\model\publishing\event\RemoteDeliveryCreatedEvent;
+
+class RegisterPublishedDeliveryWebhook extends ScriptAction
+{
+    use ServiceManagerAwareTrait;
+
+    private const DEFAULT_HTTP_METHOD = 'GET';
+
+    const ACCEPTED_HTTP_METHOD = [
```

Comment:
Please add visibility
https://www.php-fig.org/psr/psr-12/#43-properties-and-constants